src/view/terminalview.cpp

```diff
@@ -0,0 +1,77 @@
+#include "terminalview.h"
+#include <iostream>
+
+void TerminalView::render(World & world) {
+    std::string table = createTable(world);
+    std::cout << table << std::endl;
+}
+
+std::string TerminalView::createTable(World & world) {
+    std::string outputString;
+
+    outputString += "—————————————————————————————————————————\n"
+                    "Food:\n"
+                    "—————————————————————————————————————————\n";
+
+    std::vector<Food *> food = world.getFood();
+
+    for (int i = 0; i < food.size(); ++i) {
+        outputString += stringifyFood(food[i]);
+    }
+
+    outputString += "—————————————————————————————————————————\n"
+                    "Individuals:\n"
+                    "—————————————————————————————————————————\n";
+
+    std::vector<Individual *> individuals = world.getIndividuals();
+    for (int i = 0; i < individuals.size(); ++i) {
+        outputString += stringifyIndividual(individuals[i]);
+    }
+
+    return outputString;
+};
+
+std::string TerminalView::formatNumberSpacing(int number, int maxLen) {
```

Comment:
das könnte eine generelle Methode sein mit dem Namen einfach nur `padLeft` oder so, und statt "number" kann es eigentlich einen beliebigen parameter nehmen?

Reply:
No, it only takes numbers. It doesn't make sense to overload the method since there is nothing else other than numbers being padded. I could change the name of the function but that seems like kind of a trivial change

Reply:
I mean
you could change it to a very generic function taking one arbitrary parameter, the supposed string length, and whether to align it le/mi/ri.
Mayber later we add other things where we need it to work with other things than numbers.

Reply:
also, please be aware that "Number" mathematically refers to any kind of number; not only "natural numbers" like ints, but also floating point, irrational and complex numbers.
that's not really an issue, it just might confuse people that it only ever takes an int, and not, e.g., a float

Reply:
I don't think that's really needed right now so I shouldn't make it unnecessarily complex

Reply:
commit 798385a8b2e10d06f0d4c2917cf2279194084543

Reply:
Hm, maybe I could avoid the len variable by inlining it, thoughts?

Reply:
commit fad6b2237c5fa881ec634659fec7c9f5b55d14b3